sample_tpl/inserter.tpl

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
Consider adding a test inserting into `test_2` to check if the case where there are multiple indices is well handled.